app/views/layouts/application.html.haml

```diff
@@ -2,9 +2,11 @@
 %html
   %head
     %meta{content: 'text/html; charset=UTF-8', http: {equiv: "Content-Type"}}/
-    %title Boilerplate
+    %title Kimchen FoodTruck
     = csrf_meta_tags
     = stylesheet_link_tag    'application', media: 'all', data: {'turbolinks-track' => 'reload'}
     = javascript_include_tag 'application', data: {'turbolinks-track' => 'reload'}
+    %link(href="https://cdn.jsdelivr.net/npm/tailwindcss/dist/tailwind.min.css" rel="stylesheet")
```

Comment:
Adding tailwind in this story is out of scope

Reply:
true

Reply:
done